meinberlin/apps/votes/templates/meinberlin_votes/token_export_dashboard.html

```diff
@@ -0,0 +1,23 @@
+{% extends "a4dashboard/base_dashboard_project.html" %}
+{% load i18n %}
+
+{% block dashboard_project_content %}
+    <h1 class="u-first-heading">{% translate 'Download voting codes' %}</h1>
+
+    <p class="form-hint u-bottom-divider">{% blocktranslate %}For voting in the third phase, all participants need their own voting code. Here you can download the codes in packages of {{ export_size }} per excel file. To generate the codes, please contact {{ contact_email }}{% endblocktranslate %}</p>
+    <p class="u-bottom-divider">{% blocktranslate %}Number of generated codes: {{ number_of_module_tokens }}.{% endblocktranslate %}</p>
+    <ul class="u-list-reset">
+        {% if token_export_url %}
+            {% for page in token_export_iterator %}
+            <li class="u-bottom-divider u-space-between">
+                <div class="" id="id-export-tokens-hint">
+                    {% translate " code package no." %} - {{ page }}
```

Comment:
Do we still want the - here or is that a left over?

Reply:
Yes, most likely not, will check the wording and fix!

Reply:
@Rineee Because of the renaming after, I added that fix to the last commit. :woman_shrugging: